### PR TITLE
Tabs pattern: Add aria-orientation reference to keyboard note

### PR DESCRIPTION
**Pull request by aardrian**
Addresses concerns raised in #976 .

***
<a href="https://pr-preview.s3.amazonaws.com/aardrian/aria-practices/pull/1193.html" title="Last updated on Nov 11, 2019, 9:43 AM UTC (1d2a6e7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria-practices/1193/a2b4df9...aardrian:1d2a6e7.html" title="Last updated on Nov 11, 2019, 9:43 AM UTC (1d2a6e7)">Diff</a>

### CODE REVIEW

File: aria-practices.html [hunk 1]
```diff
@@ -2625,7 +2625,7 @@ <h4>Keyboard Interaction</h4>
             For additional guidance, see <a href="#kbd_selection_follows_focus"></a>.
           </li>
           <li>
-            If the tabs in a tab list are arranged vertically:
+            If the tabs in a tab list are arranged vertically (the property <a href="#aria-orientation" class="property-reference">aria-orientation</a> is set to <code>vertical</code>):
```
Comment (ZoeBijl) [inline aria-practices.html]: Why is the added text in parenthesis?

Comment (aardrian) [reply aria-practices.html]: That works too.

Comment (ZoeBijl) [reply aria-practices.html]: Could you please push those changes to your branch?

Comment (aardrian) [reply aria-practices.html]: I think I just did.

Comment (ZoeBijl) [reply aria-practices.html]: Excellent, thank you very much.

**Comment by ZoeBijl** [comment]
> My only concern is that merging this will make the tabs pattern inconsistent with other patterns that include aria-orientation.

Why are they now inconsistent @mcking65?

**Comment by aardrian** [comment]
@ZoeBijl @mcking65 

> > My only concern is that merging this will make the tabs pattern inconsistent with other patterns that include aria-orientation.
>
> Why are they now inconsistent @mcking65?

No other pattern currently on APG uses aria-orientation in its examples, 6 reference it in their overview. Only menu and toolbar have similar language and I am happy to make a PR for those. More detail and probably a good place to continue the discussion at https://github.com/w3c/aria-practices/issues/976#issuecomment-552462651.

**Pull request merged**